cdap-ui/app/cdap/components/PipelineList/EmptyList/EmptySearch.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as React from 'react';
+import T from 'i18n-react';
+import Heading, { HeadingTypes } from 'components/Heading';
+
+interface IEmptySearchProps {
+  onClear: () => void;
+  search: string;
+}
+
+const PREFIX = 'features.PipelineList.EmptyList.EmptySearch';
+
+const EmptySearch: React.SFC<IEmptySearchProps> = ({ onClear, search }) => {
```

Comment:
nit: Can we use `EmptyMessageContainer` here?

Reply:
addressed